tests/disas/many-return-values.wat

```diff
@@ -0,0 +1,43 @@
+;;! target = "x86_64"
+;;! test = "compile"
+
+(module
```

Comment:
Could you add this as a `tests/misc_testsuite/*.wast` test as well so it gets tested on all architectures automatically? I figure it's ok to avoid running the test, just having it there compiled everywhere seems good